frontend/packages/core/src/link.tsx

```diff
@@ -4,15 +4,23 @@ import { Link as MuiLink } from "@material-ui/core";
 import styled from "styled-components";
 
 const StyledLink = styled(MuiLink)`
-  padding-left: 5px;
+  ${({ ...props }) => `
+  padding-top: 10px;
+  font-size: 16px;
```

Comment:
not sure whether it's a good idea to change the font-size and padding for all links - can we just do this for our own link (the one in experimentation details view)? There are other places in Clutch that use `Link`s and they assume a lower font-size I guess?

Reply:
Well, Derek made changes here so I think he is the best person to ask. 
@dschaller what do you suggest here?

Reply:
If Derek is OK you get +1 from me but I'm afraid that this will break the formatting of some of the existing links including the ones that we have in squeeze tests.

Reply:
IMO font-size should be configurable on the caller site. Padding could also be added on the caller-site as needed since there are valid cases for when we will probably do not want to have any padding

Reply:
I didn't change the font size or padding top. The changes I made should have been backwards compatible with existing links since I only set a width if one is set as a prop. When I was testing this I didn't see a lot of things using links. At a min we shouldn't drop padding-left since this ensures links don't run into the end of another word.

Reply:
added `padding-left` back